packages/ia-components/sandbox/details/Tabby.jsx

```diff
@@ -60,22 +58,22 @@ export default class Tabby extends IAReactComponent {
           <a
             id={`tabby-${this.props.id}-finder`}
             className={`stealth${this.props.default ? ' tabby-default-finder' : ''}`}
-            identifier={this.itemid}
             href={this.state.href}
             onClick={this.onClick}
           >
             { this.props.abbreviatedText
               ? (
                 <>
-                  <I18nSpan className="tabby-text hidden-xs-span" en={this.props.text}/>
-                  <I18nSpan className="tabby-text visible-xs-span" en={this.props.abbreviatedText}/>
+                  <I18nSpan className="tabby-text hidden-xs-span" en={this.props.text} />
+                  <I18nSpan className="tabby-text visible-xs-span" en={this.props.abbreviatedText} />
                 </>
               )
-              : <I18nSpan className="tabby-text" en={this.props.text}/>
+              : <I18nSpan className="tabby-text" en={this.props.text} />
             }
           </a>
         </div>
       </div>
     );
   }
 }
+// Code inspection by Mitra 2019-12-29 exc html compare
```

Comment:
linter didn't bark without the new line at the end?

Reply:
Yes - usually does, but this comment was added to note that I'd done the linting :-)